pop-cli-for-appchains/guides/running-on-paseo-locally.md

```diff
@@ -209,32 +194,50 @@ Secret phrase:       innocent throw harsh wild example reflect sausage leopard l
 
 > Save the secret phrase in a secure manner.
 
-## Creating the chain spec for the parachain
+We now have all the accounts we need and we can start prepare our chain!
 
-Let's create a chain spec for our parachain:
+## Setting up the Chain
 
+For the sake of this exercise, let's create a new chain project:
 ```
-cd my-parachain
+pop new parachain my-chain
 ```
+> The folder includes a `network.toml` file which can be ignored. This is to launch a network with the chain already onboarded.
 
-Build your parachain for release:
 
-```
-pop build --release
-```
+### Creating the chain spec
 
-Create the chain spec:
+The chain specification holds all the information the node requires to start or sync with the chain's network.
 
-```
-pop build spec --release --id 2000 --type local --relay paseo-local --protocol-id my_parachain
-```
+Let's generate a chain spec:
 
-This will output the following:
+<figure><img src="../.gitbook/assets/buildspec.gif" alt="pop build spec"><figcaption><p>pop build spec</p></figcaption></figure>
 
-* Your parachain's chain specification file
-  * The plain text and the raw version e.g. `chain-spec.json` and `chain-spec-raw.json`
-* Your parachain's initial genesis state e.g. `para-2000-genesis-state`
-* Your parachain's Wasm runtime e.g. `para-2000.wasm`
+```
+cd my-chain
+pop build spec --profile release --id 2000 --type local --relay paseo-local --protocol-id my_chain --chain local --genesis-state --genesis-code
+```
+```bash
```

Comment:
Do we still need this output if we have the gif?

Reply:
For copy pasting it can still be handy perhaps.

Reply:
You are right, removed the command and output.